api/util/imagery/imagery.go

```diff
@@ -222,7 +224,14 @@ func ImageFromCombination(datasetDir string, bandFileMapping map[string]string,
 			filePaths = append(filePaths, path.Join(datasetDir, bandFileMapping[bandLabel]))
 		}
 
-		image, err := ImageFromBands(filePaths, bandCombo.Ramp, bandCombo.Transform, imageScale, options...)
+		var imageRamp []uint8
```

Comment:
This can be simplified to:
`
// default
imageRamp := bandCombo.Ramp

if ramp != "" {
      imageRamp = GetColorRamp(ramp)
}
`